airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -565,6 +565,9 @@
   icon: freshdesk.svg
   sourceType: api
   releaseStage: generally_available
+  allowedHosts:
+    hosts:
+      - "${domain}"
```

Comment:
Please add an explicit list of allowed hosts

Reply:
Actually, this might be okay - here is the definition of how we have the user provide their domain: 
```
      "domain": {
        "type": "string",
        "description": "Freshdesk domain",
        "title": "Domain",
        "examples": ["myaccount.freshdesk.com"],
        "pattern": "^[a-zA-Z0-9._-]*\\.freshdesk\\.com$"
      },
 ```
 
 So it seems similar to enforcing `*.freshdesk.com`, but restricted to what the user put in.

Reply:
changed to `*.freshdesk.com`. Now I think it more safe then to put whatever a client provide us with